test/functional/api/vim_spec.lua

```diff
@@ -1036,5 +1036,33 @@ describe('api', function()
       eq(expected, nvim("list_uis"))
     end)
   end)
-
+  describe('nvim_get_keyboard_mods_state', function()
+    it('on linux either xlib11 is available or not', function()
+      if os_name() == 'unix' then
+        local status, rv = pcall(nvim, "get_keyboard_mods_state")
+        if status == true then
+          eq(rv, {CapsLock=false, NumLock=false, ScrollLock=false})
+        else
+          eq("Unable to load ", string.match(rv, "Unable to load *"))
+        end
+      end
+    end)
+    it('on windows there is always an acceptance', function()
+      if os_name() == 'windows' then
+        eq(nvim('get_keyboard_mods_state'),
+                {CapsLock=false, NumLock=false, ScrollLock=false})
```

Comment:
This also has wrong alignment, dictionary is not an argument to `nvim()`.

Reply:
No, nvim is a function which calls the function `nvim_get_keyboard_mods_state`, whose output is checked by eq, the dictionary is outside the `nvim` function.
I will replace the `nvim` although, due to the next review